.github/workflows/formatting.yml

```diff
@@ -0,0 +1,38 @@
+name: Formatting (black & isort)
+on:
+  push:
+    branches: master
+    paths:
+      - '**.py'
```

Comment:
Can we run this always?

I am not sure whether we have executables without `.py` suffix ...

Reply:
Just had a quick check, doesn't look like it in this repo